book.toml

```diff
@@ -14,6 +14,7 @@ class = "bob"
 [output.html]
 curly-quotes = true
 additional-js = ["ga4.js"]
+additional-css = ["ga4.css"]
```

Comment:
Please call this something like `svgbob.css` instead — the name `ga4` is short for Google Analytics 4 since that's what the JavaScript file implements :-)

Reply:
I renamed the file and pushed to your branch. I'll merge this next.